tycho-common/src/dto.rs

```diff
@@ -1490,9 +1490,22 @@ impl From<TracedEntryPointRequestResponse> for DCIUpdate {
     }
 }
 
-pub struct AddEntrypointRequestBody {
-    pub entrypoints_with_tracing_data: Vec<(String, Vec<EntryPointWithTracingParams>)>,
+#[derive(Serialize, Deserialize, Debug, Default, PartialEq, ToSchema, Eq, Clone)]
+pub struct AddEntryPointRequestBody {
+    #[serde(default)]
     pub chain: Chain,
+    #[schema(value_type=String)]
+    #[serde(default)]
+    pub block_hash: Bytes,
+    /// The map of component ids to their tracing params to insert
+    pub entry_points_with_tracing_data: Vec<(String, Vec<EntryPointWithTracingParams>)>,
```

Comment:
Picky comment, but we have a type in `tycho-common` that I'd rather use for `ComponentId`;
```rust 
pub type ComponentId = String;
```

Makes it easier to identify that it's not any string, but a `ComponentId` one - so I like to use them, specially on external interfaces